angular-dependancy-injection/dependancy-injection/src/app/expense-table/expense-table.component.html

```diff
@@ -0,0 +1,100 @@
+<div class="container">
+    <br>
+    <h1 class="text-center"><strong>Expenses Table - {{current_year}}</strong></h1>
+    <br>
+    <div class="container card-container custom-class">
```

Comment:
Use NgFor to render Months

Reply:
Done, will push with other changes at once.